chat-plugins/modlog.js

```diff
@@ -280,15 +318,30 @@ exports.commands = {
 		let addModlogLinks = Config.modloglink && (!hideIps || (targetRoom && targetRoom.isPrivate !== true));
 		// Let's check the number of lines to retrieve or if it's a word instead
 		let lines = 0;
+		let more = false;
+		let moreIncrement = 0;
+		if (target.includes('LINE INCREMENT=')) {
```

Comment:
You do realize that `lines=` is hard to type accidentally, right?

I just mean that `/modlog lines=52` is going to be unambiguous, while `/modlog 52` isn't (maybe you're searching for times people have mentioned "52")